### PR TITLE
Minor fix on save extra AOVs (originally #1865)

### PR DESCRIPTION
**Pull request by oktomus**
Original PR: #1865

### CODE REVIEW

File: src/appleseed/renderer/modeling/frame/frame.cpp [hunk 1]
```diff
@@ -706,7 +706,7 @@ namespace
     {
         bool success = true;
 
-        if (extension != ".exr")
+        if (extension != ".exr" && extension != ".EXR")
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/frame/frame.cpp]: That's not robust enough (wouldn't work with ".Exr" for instance). I suggest converting the extension to lower case before doing the check.

Comment (oktomus) [reply src/appleseed/renderer/modeling/frame/frame.cpp]: Ok, then we should also do that in write_image

Comment (dictoon) [reply src/appleseed/renderer/modeling/frame/frame.cpp]: It's already properly done in `write_image()`:

    string extension = lower_case(bf_file_path.extension().string());

Comment (dictoon) [reply src/appleseed/renderer/modeling/frame/frame.cpp]: That said, the problem appears to also exist in `write_main_image()`, maybe that's what you meant.

Comment (oktomus) [reply src/appleseed/renderer/modeling/frame/frame.cpp]: No it wasn't, I haven't seen the lower_case code. But thanks for pointing out this, I will fix that

**Comment by dictoon** [comment]
Thanks!

**Pull request merged**